content/doc/book/managing/plugins.adoc

```diff
@@ -225,6 +225,67 @@ The configuration(s) created by the disabled plugin behave as if the plugin
 were uninstalled, insofar that they result in warnings on boot but are
 otherwise ignored.
 
+==== Using the Jenkins CLI
+
+It is also possible to enable and disable plugins using the <<cli#,Jenkins CLI>>.
+Since Jenkins 2.136, administrators may invoke the `enable-plugin` command to 
+re-enable again a collection of plugins which were previously disabled and since
+Jenkins 2.151, the `disable-plugin` command is included, which provides a command
+to disable installed plugins.
+
+The `enable-plugin` command receives as parameter a list of plugins to be enabled.
+If any of the plugins have one or more dependencies, these plugins will be enabled
```

Comment:
Maybe these dependencies will be enabled as well? The second 'plugins' might be confusing because it's not clear if you are referring to the provided list or the dependencies.

Reply:
"plugins and their transitive dependencies" ?

Reply:
I think "plugins and their transitive dependencies" is better